src/base/io/utilities/createEmptyFields.m

```diff
@@ -0,0 +1,90 @@
+function model = createEmptyFields(model,fieldNames, fieldDefinitions)
```

Comment:
This function needs minor changes in the documentation in order to generate properly.